scripts/lib/settings.py

```
@@ -91,4 +91,4 @@ class McbSettings(BaseMcbSettings):
     )
 
 
-__all__ = ["BaseCommandSettings", "BaseMcbSettings", "McbSettings"]
+__all__ = ["BaseMcbSettings", "BaseCommandSettings", "McbSettings"]
```

Ruff (0.16.2)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)